src/FreeScribe.client/UI/LoadingWindow.py

```diff
@@ -133,5 +133,6 @@ def destroy(self):
             # Enable the parent window
             if self.parent:
                 self.parent.wm_attributes('-disabled', False)
-            self.progress.stop()
+            if self.progress.winfo_exists():
+                self.progress.stop()
             self.popup.destroy()
```

Comment:
Actually, you just check if it exists to stop it, either way it will destroy it.   I agree with sourcery, would make more sense I think to have it in the block that you check if it exists before you stop it?  Would there be a case that you would destroy it if it didn't exist? Might not be critical but it is cleaner.

Reply:
https://github.com/ClinicianFOCUS/FreeScribe/pull/123/commits/27c6b0fc6fff88aa8ce120a4892fb8ae8ed34d1b